scripts/city.js

```diff
@@ -0,0 +1,22 @@
+import { getLocalTime, dewPointCelsius, findWindDirection } from './helpers.js'
+export class CityInfo {
```

Comment:
use blank line here to format code
```suggestion

export class CityInfo {
```